tests/jeos/firewall_enabled.pm

```diff
@@ -0,0 +1,20 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self           = shift;
```

Comment:
and yet another nitpick: you don't need to set $self if you don't need it in the run function